bookmarklet.js

```diff
@@ -0,0 +1,18 @@
+var module;
+var script = document.createElement('script');
+script.onload = function() {
+  var stats = new MemoryStats();
+
+    stats.domElement.style.position = 'fixed';
+    stats.domElement.style.right        = '0px';
+    stats.domElement.style.bottom       = '0px';
+
+    document.body.appendChild( stats.domElement );
+
+    requestAnimationFrame(function rAFloop(){
+        stats.update();
+        requestAnimationFrame(rAFloop);
+    });
+};
+script.src = "https://rawgit.com/paulirish/memory-stats.js/master/memory-stats.js";
```

Comment:
As I understand it, that is not a problem because the bookmarklet.js file is also loaded from rawgit.com (to get correct content type).